src/main/java/mate/academy/springintro/model/User.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.springintro.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Data
+@Entity
+@SQLDelete(sql = "UPDATE users SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted = false")
+@Table(name = "users")
+public class User {
```

Comment:
Add User entity with the following fields:
id (Long, PK)
email (String, not null, unique)
password (String, not null)
firstName (String, not null)
lastName (String, not null)
shippingAddress (String)